package/crds/ec2.aws.jet.crossplane.io_securitygroups.yaml

```diff
@@ -183,12 +176,7 @@ spec:
                         self:
                           type: boolean
                         toPort:
-                          format: int64
-                          type: integer
-                      required:
```

Comment:
Why are these not required anymore?

Reply:
They're not marked as `"required": true` in the cli schema. I generated preview CRDs and noticed the same change in some of the other resources too.

Reply:
Could you note this case [here](https://github.com/crossplane/terrajet/issues/262) or open an issue so that we can investigate it? Ideally, we shouldn't lose any information due to conversions done by TF CLI or us. Noting it down somewhere would help us know what to fix without trying again.